packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -235,6 +248,12 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
     },
     button__disabled: {
         ...applyDisabledState(),
+        "& $button_beforeContent": {

```

Comment:
can you chain these? `& $button_beforeContent, & $button_afterContent`

Reply:
See my above comment to @chrisdholt - not sure why but chaining them doesn't allow `ButtonHighlight` to be passed.